### PR TITLE
Add Migration to populate standard products

### PR DESCRIPTION
**Pull request by pylipp**
Merge after #586 only.

### CODE REVIEW

**Comment by pylipp** [comment]
[2.0 ASSORT: Database migrations](https://trello.com/c/YkzqLWwm/1398-20-assort-database-migrations)

File: db/migrations/20240523182223_populate_standard_products_table.php [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+use Phinx\Migration\AbstractMigration;
+
+class PopulateStandardProducts extends AbstractMigration
+{
+    public function up(): void
+    {
+        $table = $this->table('standard_product');
+        $this->execute('
+INSERT INTO standard_product
+(gender_id,name,size_range_id,category_id,version,added_by)
```
Comment (HaGuesto) [inline db/migrations/20240523182223_populate_standard_products_table.php:12]: please add an id

Comment (HaGuesto) [reply db/migrations/20240523182223_populate_standard_products_table.php:12]: please add added_on

**Review by HaGuesto — APPROVED** [review]
functional and code review passed

**Pull request merged**